tests/expectations/metrics/query_metrics/test_query_metrics.py

```diff
@@ -8,6 +8,28 @@
 )
 
 
+@pytest.mark.unit
+def test_query_template_get_query_function_with_str():
```

Comment:
Unittest added to `Query Metrics` testing the str replacement from template, which is used by the current Expectation